mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -34,6 +34,43 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
```

Comment:
Nice, I like that it extends HasRegParam.  When you update it, can you please make the doc specify more about the parameter (range, plus which end of the range corresponds to L1 vs. L2)?